src/programs/someone-tag.ts

```diff
@@ -8,90 +16,64 @@ const QUESTION_SHEET_ID: string =
   "1eve4McRxECmH4dLWLJvHLr9fErBWcCGiH94ihBNzK_s";
 
 @Command({
-  event: DiscordEvent.MESSAGE,
-  trigger: "@someone",
-  allowedRoles: ["Seek Discomfort"],
-  channelNames: ["chat", "chat-too", "4th-chat", "chat-v"],
-  description: "This handler is for the someone command.",
+  event: DiscordEvent.SLASH_COMMAND,
+  root: "someone",
+  description: "Ping a random memeber.",
+  options: [
+    {
+      type: ApplicationCommandOptionType.Boolean,
+      name: "Online",
```

Comment:
The name must be lowercase (otherwise it results in a runtime error when starting the bot):

```suggestion
      name: "online",
```